portality/templates/application_form/editorial_form_fields.html

```diff
@@ -11,6 +11,21 @@ <h2><span data-feather="settings" aria-hidden="true"></span> Administrative</h2>
     <button class="button button--primary" type="button" id="open_withdraw_reinstate" data-toggle="modal" data-target="#modal-withdraw_reinstate">{{ withdraw_reinstate }}</button>
 {% endif %}
 
+{% set fs = formulaic_context.fieldset("oa_start") %}
+{% if fs %}
+<fieldset>
+    <h3 style="margin: 0 0 .75rem 0;">{{ fs.label }}</h3>
+    <div class="row">
+    {% for f in fs.fields() %}
+        <div class="col-xs-6">
+            {% set field_template = f.template %}
+            {%  include field_template %}
+        </div>
+    {% endfor %}
+    </div>
+</fieldset>
+{% endif %}
```

Comment:
Is this section necessary now that the field (in phase 2) is part of the main application form?

Reply:
No, it is not. removed. Thank you!